config/initializers/simple_form_bootstrap.rb

```diff
@@ -45,6 +45,19 @@
   config.input_field_error_class = 'is-invalid'
   config.input_field_valid_class = 'is-valid'
 
+  # Configure layout of date input, time input, and datetime input.
+  SimpleForm::Inputs::DateTimeInput.default_options.tap do |options|
+    options[:date_separator] = '<div class="mx-1"></div>'
+    options[:time_separator] = '<div class="mx-1">:</div>'
+    options[:datetime_separator] = '<div class="col-2 mw-100 flex-shrink-1 text-center mt-1">&mdash;</div>'
```

Comment:
💅 IMHO the `datetime_separator` could get a narrow space by using `col-1` instead. `.col-2` in combination with the `&mdash;` is too much.

Reply:
The reason the `&mdash;` is `col-2` is to force it to wrap to the second line when on mobile.  The date inputs take up 10 (3 + 5 + 2) columns, and the spacers between them take up a half column or so.  Thus if the `&mdash;` was `col-1` it would stay on the first line when wrapping, which would shrink the date inputs and expand the time inputs (the opposite of what we need).

I played around with **many** different `col-X` and spacer combinations and this was the best compromise I could find for both desktop and mobile.

Reply:
Thank you for the explanation, didn't consider these options. 

What do you think about this change, because the row get wrapped (`.flex-wrap .flex-md-nowrap`) thus hide the `datetime_separator` on this viewport.

```diff
-    options[:datetime_separator] = '<div class="col-2 mw-100 flex-shrink-1 text-center mt-1">&mdash;</div>'
+    options[:datetime_separator] = '<div class="col-auto d-none d-md-block mt-1">&mdash;</div>'
```

Reply:
Hmm... to me a `col-2` `&mdash;` is more of a problem when the inputs aren't wrapped, i.e. on displays >= `md`.  When the inputs aren't wrapped, the `col-2` steals space from all the inputs.  When the inputs are wrapped, the `col-2` only steals space from the time inputs, which are already overly large.

We could possibly remove the `&mdash;` altogether, but I don't know if that would be acceptable or not.

Also, while an `&mdash;` with `d-none d-md-block` works on mobile, it won't affect the horizontal form layout on desktop (where wrapping can also occur).